.github/workflows/snyk_sca_scan.yaml

```diff
@@ -1,14 +1,13 @@
 name: Snyk Software Composition Analysis Scan
 # This git workflow leverages Snyk actions to perform a Software Composition 
-# Analysis scan on our Opensource libraries upon Pull Requests to Master & 
+# Analysis scan on our Opensource libraries upon Pull Requests to
 # Develop branches. We use this as a control to prevent vulnerable packages 
```

Comment:
```suggestion
# Analysis scan on our Opensource libraries upon Pull Requests to the
# "develop" branch. We use this as a control to prevent vulnerable packages 
```